### PR TITLE
Fix issue #623: moving icons to a non-existing path

### PR DESCRIPTION
**Pull request by eldarkg**

### CODE REVIEW

**Comment by eldarkg** [comment]
@probonopd Can you check this PR?

**Comment by probonopd** [comment]
Thanks @eldarkg. The many whitespace corrections make it hard to see what's the real fix. What is it?

File: src/shared.c [hunk 1]
```diff
@@ -163,15 +163,15 @@ void move_icon_to_destination(gchar *icon_path, gboolean verbose)
...
+
+        // FIXME: The following sizes are taken from the hicolor icon theme.
         // Probably the right thing to do would be to figure out at runtime which icon sizes are allowable.
         // Or could we put our own index.theme into .local/share/icons/ and have it observed?
         if((w != h) || ((w != 16) && (w != 24) && (w != 32) && (w != 36) && (w != 48) && (w != 64) && (w != 72) && (w != 96) && (w != 128) && (w != 192) && (w != 256) && (w != 512))){
             fprintf(stderr, "%s has nonstandard size w = %i, h = %i; please fix it\n", icon_path, w, h);
         } else {
             g_free(dest_dir);
-            dest_dir = g_build_path("/", g_get_user_data_dir(), "/icons/hicolor/", g_strdup_printf("%ix%i", w, h), "/apps", NULL);
+            dest_dir = g_build_path("/", g_get_user_data_dir(), "/icons/hicolor/", g_strdup_printf("%ix%i", w, h), "/apps/", NULL);
```
Comment (eldarkg) [inline src/shared.c]: Here

File: src/shared.c [hunk 2]
```diff
@@ -128,28 +128,28 @@ char *get_thumbnail_path(const char *path, char *thumbnail_size, gboolean verbos
...
  * the icon as part of their directory name, as specified in the theme.index
  * See https://github.com/AppImage/AppImageKit/issues/258
  */
 
 void move_icon_to_destination(gchar *icon_path, gboolean verbose)
 {
     // FIXME: This default location is most likely wrong, but at least the icons with unknown size can go somewhere
-    gchar *dest_dir = g_build_path("/", g_get_user_data_dir(), "/icons/hicolor/48x48/apps", NULL);;
-    
+    gchar *dest_dir = g_build_path("/", g_get_user_data_dir(), "/icons/hicolor/48x48/apps/", NULL);
```
Comment (eldarkg) [inline src/shared.c]: Here

**Comment by eldarkg** [comment]
@probonopd, See the last comment. Adds a `\` after `apps`

**Comment by probonopd** [comment]
Thank you.

**Pull request merged**